library/std/src/sys/unix/ext/io.rs

```diff
@@ -58,6 +84,19 @@ pub trait IntoRawFd {
     /// This function **transfers ownership** of the underlying file descriptor
     /// to the caller. Callers are then the unique owners of the file descriptor
     /// and must close the descriptor once it's no longer needed.
+    ///
+    /// # Example
+    ///
+    /// ```no_run
+    /// use std::fs::File;
+    /// use std::os::unix::io::IntoRawFd;
+    ///
+    /// fn main() -> std::io::Result<()> {
```

Comment:
Why not remove the `-> std::io::Result<()>`?

Reply:
It allows to use the `?` operator which is the recommended way. :)

Reply:
Oh, I mean something like this which still have `?`.
```rust
use std::fs::File;

File::open("foo.txt")?;
Ok(())
```

Reply:
I'm not sure it works. But it's worth a try!